cfme/common/vm.py

```diff
@@ -21,6 +21,7 @@
 from cfme.utils.rest import assert_response
 from cfme.utils.timeutil import parsetime
 from cfme.utils.update import Updateable
+from cfme.utils.version import current_version
```

Comment:
**REQUIRED**
We are trying to avoid using of `current_version`. Just compare `appliance.version`.